terraform/modules/dns_logging/resolver_logging.tf

```diff
@@ -3,6 +3,9 @@ module "dns_resolver_logs_bucket" {
   bucket          = "${var.stack_description}-query-resolver-logs"
   aws_partition   = var.aws_partition
   expiration_days = 930 # 31 days * 30 months = 930 days
+  # need to exclude bucket policy otherwise we get access denied errors from Route53
+  # query resolver logging configuration
```

Comment:
expand this comment to add:
```The bucket's default encryption setting is still enforced to ensure "encryption at rest"```

Reply:
aecef1a3c82eef3bf00de72856f702ee3aadc29c